### DISCUSSION TITLE
Custom Login UI

### DISCUSSION THREAD

**Discussion by Tylerchristensen100** [zitadel/zitadel#8985] [Q&A] [answered]
I am building an app but it has language requirements zitadel doesn't meet.  

I have looked through the Custom UI Docs and the https://github.com/zitadel/typescript docs but I can't find a way to login to an account without an Auth Token.

I just need to build a simple username and password login.  Is there a way to just do it on the frontend or is a backend required so a Service User token can be used?

**Comment by hifabienne** [comment]
@peintnermax can you help here?

**Comment by peintnermax** [comment, marked as answer]
Hi @Tylerchristensen100,

Can you provide more details on your language requirements? 
Yes, our new session API requires a service user token and a confidential environment to create or update sessions.
Which technologies are you using for your frontend? Most frameworks offer a Backend for frontend layer, or it can easily be implemented using serverless functions.

**Reply by Tylerchristensen100** [reply]
My backend is written in Go with a Flutter frontend.

Thank you for your timely response.  I was mostly just unsure from the documentation if it was supposed to be frontend only, or if a backend needed to be involved.  

But, with this clarification, the documentation makes sense.

Thanks!
